kitsune/sumo/tests/test_email_utils.py

```diff
@@ -132,3 +133,30 @@ def test_styles_inlining(self):
             for m in msg:
                 tag = '<a href="https://%s/test" style="color:#000">Hyperlink</a>'
                 self.assertIn(tag % Site.objects.get_current().domain, str(m.message()))
+
+
+class SendMessagesTests(TestCase):
+
+    @patch("kitsune.sumo.email_utils.mail")
+    def test_when_messages_have_valid_email(self, mock_mail):
+        from_email = "notifications@support.mozilla.org"
+        messages = [
+            EmailMultiAlternatives("Test", "Testing", from_email, ["ringo@beatles.org"]),
+            EmailMultiAlternatives("Test", "Testing", from_email, ["george@beatles.org"]),
+            EmailMultiAlternatives("Test", "Testing", from_email, ["paul@beatles.org"]),
+        ]
+        send_messages(messages)
+        send_messages_mock = mock_mail.get_connection().__enter__().send_messages
+        send_messages_mock.assert_called_once_with(messages)
+
+    @patch("kitsune.sumo.email_utils.mail")
+    def test_when_message_has_invalid_email(self, mock_mail):
+        from_email = "notifications@support.mozilla.org"
+        messages = [
+            EmailMultiAlternatives("Test", "Testing", from_email, ["ringo@beatles.org"]),
+            EmailMultiAlternatives("Test", "Testing", from_email, ["george.@beatles.org"]),
```

Comment:
This is actually a valid address of gmail

Reply:
Oh, I didn't know that, thanks Tasos. I'll adjust this PR accordingly.